en/docs/reference/connectors/cdc-inbound-endpoint/cdc-inbound-endpoint-config.md

```diff
@@ -0,0 +1,182 @@
+.tg th{font-family:Arial, sans-serif;font-size:20px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;}
+.tg .tg-0pky{border-color:inherit;text-align:left;vertical-align:top}
+</style>
+<table class="tg">
+  <tr>
+    <th class="tg-0pky">Parameter</th>
+    <th class="tg-0pky">Default Value</th>
+    <th class="tg-0pky">Description</th>
+  </tr>
+  <tr>
+    <td class="tg-0pky">interval</td>
+    <td class="tg-0pky">-</td>
+    <td class="tg-0pky">The polling interval for the inbound endpoint to execute each cycle. This value is set in milliseconds.</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">coordination</td>
+    <td class="tg-0pky">true</td>
+    <td class="tg-0pky">This optional property is only applicable in a cluster environment. In a clustered environment, an inbound endpoint will only be executed in worker nodes. If set to <code>true</code> in a cluster setup, this will run the inbound only in a single worker node. Once the running worker is down, the inbound starts on another available worker in the cluster. By default, coordination is enabled.</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">sequential</td>
+    <td class="tg-0pky">true</td>
+    <td class="tg-0pky">Whether the messages need to be polled and injected sequentially or not. By default this is set to <code>true</code>.</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">connector.name</td>
+    <td class="tg-0pky">-</td>
+    <td class="tg-0pky">Unique name for the connector. If not present, inbound endpoint name is considered as the connector name. </td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">snapshot.mode</td>
+    <td class="tg-0pky">initial</td>
+    <td class="tg-0pky">Specifies the criteria for running a snapshot when the connector starts. Possible settings are: always, initial, initial_only, schema_only, no_data, recovery, when_needed, configuration_based, custom.</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">connector.class</td>
+    <td class="tg-0pky">-</td>
+    <td class="tg-0pky">The name of the Java class for the connector.<br>
+Example: <br>
+For MySQL database, <code>io.debezium.connector.mysql.MySqlConnector</code><br>
+For PostgreSQL database, <code>io.debezium.connector.postgresql.PostgresConnector</code><br>
+For Oracle database, <code>io.debezium.connector.oracle.OracleConnector</code><br>
+For Db2 database, <code>io.debezium.connector.db2.Db2Connector</code></td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">topic.prefix</td>
+    <td class="tg-0pky">-</td>
+    <td class="tg-0pky">Topic prefix that provides a namespace for the database server that you want Debezium to capture. The prefix should be unique across all other connectors, since it is used as the prefix for all Kafka topic names that receive records from this connector. Only alphanumeric characters, hyphens, dots and underscores must be used in the database server logical name.</td>
```

Comment:
```suggestion
    <td class="tg-0pky">Topic prefix that provides a namespace for the database server that you want Debezium to capture. The prefix should be unique across all other connectors since it is used as the prefix for all Kafka topic names that receive records from this connector. Only alphanumeric characters, hyphens, dots, and underscores must be used in the database server logical name.</td>
```